full-service/src/db/txo.rs

```diff
@@ -635,6 +636,7 @@ pub trait TxoModel {
     ///| `max_spendable_value` | The upper limit for the spendable TxOut value to filter on |                                     |
     ///| `token_id`            | The id of a supported type of token to filter on           |                                     |
     ///| `default_token_fee`   | The default transaction fee in Mob network                 |                                     |
+    ///| `only_from_subaddress`| The subaddress from which spendable txos can be sourced    | (optional)                          |
```

Comment:
The ambiguity I'm trying to avoid is that an `assigned_subaddress_b58` is some subaddress that was created (assigned) for an expected counterparty (e.g. an exchange assigning for one of their customers, where they expect to receive funds). Because an address could be used in many ways (and typically is used only for receiving), I wanted to make it clear that this address was being used _only for spending from_ during this operation.

Let me think about this one though, because it is a good point about consistency with naming.